docs/tutorial.rst

```diff
@@ -0,0 +1,23 @@
+.. _tutorial_index:
+
+Tutorial
+********
+
+This section contains step-by-step tutorials to help you get started with
+Ubuntu packaging and development. We hope the tutorials make as few assumptions
```

Comment:
Polish: Maybe we should add encouragement to readers to provide feedback how their experience was 🤔

Reply:
I probably wouldn't do that on this page. I think it would make more sense at the end of the tutorial, but I think it'd be good to rope in the community team to get their input in terms of preferred feedback channels etc before we add in anything like that. They're the ones who will ultimately "own" this documentation and might have their own ideas for that.